### ISSUE TITLE
Inaccessible color combinations in light theme

### ISSUE THREAD

**Issue by darekkay** [radix-ui/colors#41]
Hello,
from the Radix UI homepage:

> Text colors are guaranteed to pass target contrast ratios against the corresponding background colors.

I have just published a new version of [a11y-contrast](https://github.com/darekkay/a11y-contrast) and included both Radix UI palettes under [examples](https://github.com/darekkay/a11y-contrast/tree/master/examples). Because your approach is slightly different than my ["magic numbers"](https://darekkay.com/blog/accessible-color-palette/), I have temporary removed all colors apart from "backgrounds" (1 and 2) and "accessible text" (11 and 12).

I have then checked whether each background/text combination passes WCAG AA:

```bash
npx a11y-contrast examples/radix-light.json --min-ratio-4.5=9
```

This returns 283 inaccessible color combinations. Maybe your statement means that only combinations within the same color family are expected to be always accessible (I would then suggest to make it more prominent in the documentation), but I have found a couple of violations here as well:

   - amber-2 / amber-11 (4.432952188282118)
   - orange-1 / orange-11 (4.408486700021852)
   - yellow-1 / yellow-11 (4.478200938191756)

Note: The dark palette is accessible for all background/text combinations, even cross-family.

**Comment by darekkay** [comment]
Thank you for the explanation. That's fair, and I indeed want to [include APCA](https://github.com/darekkay/a11y-contrast/issues/4) in my tool as well someday. Favoring users over specifications with problematic edge cases is a good thing. However, I would suggest to put your explanation as a short disclaimer/footnote on your homepage, to make sure that everyone is aware that the color palette is not WCAG (2) compliant. WCAG is [not without faults](https://yatil.net/blog/wcag-3-is-not-ready-yet), but it is often _the_ legally-binding definition of "accessible". When a framework / library claims to be accessible, WCAG is what accessibility consultants and testers will use to verify the claim.
